app/connectors/UserManagementClient.scala

```diff
@@ -51,7 +51,7 @@ trait UserManagementClient {
         response
       }
     }.recover {
-      case Upstream4xxResponse(_, 401, _, _) => throw new InvalidCredentialsException()
+      case Upstream4xxResponse(_, 401, _, _) => throw new InvalidCredentialsException(s"Email = $email")
```

Comment:
Although this is handy, emails shouldn't be leaked into the logs. Even wrong ones are often close to being right (i.e. single letter typo, edit distance of 1)